internal/device/energy.go

```diff
@@ -23,3 +25,30 @@ func (e Energy) MicroJoules() uint64 {
 func (e Energy) String() string {
 	return fmt.Sprintf("%fJ", e.Joules())
 }
+
+// Power represents power usage as an float64 MicroWatts.
+// Use functions Watts and MicroWatts to get the power value as
+// Watts or MicroWatts
+type Power float64
+
+const (
+	MicroWatt Power = 1.0
+	MilliWatt       = 1000 * MicroWatt
+	Watt            = 1000 * MilliWatt
+)
```

Comment:
We should do something similar to this for Joules as well in a separate commit. Inspired by time.Duration